tools/markdown.py

```diff
@@ -19,45 +19,49 @@
   http://en.wikipedia.org/wiki/Markdown
 """
 
-__version__ = '1.0.1-2'  # port of 1.0.1
+__version__ = "1.0.1-2"  # port of 1.0.1
 __license__ = "GNU GPL 2"
 __author__ = [
-  'John Gruber <http://daringfireball.net/>',
-  'Tollef Fog Heen <tfheen@err.no>',
-  'Aaron Swartz <me@aaronsw.com>'
+    "John Gruber <http://daringfireball.net/>",
+    "Tollef Fog Heen <tfheen@err.no>",
+    "Aaron Swartz <me@aaronsw.com>",
 ]
```

Comment:
Again, this is a external file included in web.py and it would be better to exclude this from pep-8.

Reply:
Since we maintain its copy instead of syncing it from upstream frequently, it's better format it too.